docs/source/contributing.rst

```diff
@@ -0,0 +1,3 @@
+.. highlight:: sh
```

Comment:
Let's reverse this doc and CONTRIBUTING.rst. I would like the content in this file and a short message referring to this document from CONTRIBUTING.rst.